python/electrode2currentmap.py

```diff
@@ -279,17 +279,23 @@ def __init__(self, freq=20, dur=0.5, pulse_dur=.075/1000.,
 
         # set up the sequence
         if stimtype == 'pulsetrain':
-            interpulsegap = np.zeros(round((1/freq) / tsample) - len(pulse))
+            # Warning: using a non-integer number instead of an integer will
+            # result in an error in the future
+            interpulsegap = np.zeros(int(1/freq/tsample) - len(pulse))
```

Comment:
`int` will always chop down to the smaller integer (e.g., `int(1.8) == 1`). Is that really what we want here?